<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vertical_Selection_Panel">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="vertical_Image_Pane">
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontal_Top_Image_Pane" stretch="0">
          <property name="spacing">
           <number>60</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_Panel_Name_Image">
            <property name="font">
             <font>
              <family>MS Serif</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Images</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="folder_Path_Image">
          <property name="maximumSize">
           <size>
            <width>273</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="image_List"/>
        </item>
        <item>
         <widget class="QComboBox" name="dropdown_Sort_Image">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>Ascending Name</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Ascending Name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Descenting Name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ascending Date</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Descending Date</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Select_Image_Folder">
          <property name="text">
           <string>Select target folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vertical_Classes_Pane">
        <item>
         <layout class="QHBoxLayout" name="horizontal_Top_Classes_Pane">
          <property name="spacing">
           <number>60</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_Panel_Name_Class">
            <property name="font">
             <font>
              <family>MS Serif</family>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Classes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="folder_Path_Class">
          <property name="maximumSize">
           <size>
            <width>273</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="class_List"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontal_Bottom_Classes_Pane">
          <item>
           <widget class="QComboBox" name="dropdown_Sort_Class">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>Ascending Name</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Ascending Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Descenting Name</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Add_Class">
            <property name="text">
             <string>Add Class</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Delete_Class">
            <property name="text">
             <string>Delete Class</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Select_Class_File">
          <property name="text">
           <string>Select target file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontal_Annotation_Pane">
        <item>
         <layout class="QVBoxLayout" name="vertical_Left_Annotation_Pane">
          <item>
           <widget class="QPushButton" name="btn_Save_Annotation">
            <property name="text">
             <string>Save Annotations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Load_Annotation">
            <property name="text">
             <string>Load Annotations</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vertical_Right_Annotation_Pane">
          <item>
           <widget class="QLabel" name="lbl_Current_Annotation_FIle">
            <property name="text">
             <string>Current annotation file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="folder_Path_Annotation">
            <property name="text">
             <string>......</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_1">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="grid_Shapes_Pane">
        <item row="2" column="1" colspan="2">
         <widget class="QRadioButton" name="rad_Btn_Triangle">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Triangle</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_Panel_Shapes_Name">
          <property name="font">
           <font>
            <family>MS Serif</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Shapes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rad_Btn_Rectangle">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Rectangle</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="rad_Btn_Trapezium">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Trapezium</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="rad_Btn_Polygon">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Polygon</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBox_Polygon_Sides"/>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="lbl_Polygon_Side">
          <property name="text">
           <string>Sides</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="Canvas" name="canvas">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1198</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QLabel</extends>
   <header location="global">canvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
